luatest/server.lua

```diff
@@ -265,6 +243,28 @@ function Server:start(opts)
 
     self:initialize()
 
+    fio.mktree(self.workdir)
+    if self.datadir then
+        local ok, err = fio.copytree(self.datadir, self.workdir)
+        if not ok then
+            error(('Failed to copy %s to %s: %s'):format(self.datadir, self.workdir, err))
+        end
+        self.datadir = nil
+    end
+    local parsed_net_box_uri = uri.parse(self.net_box_uri)
+    if parsed_net_box_uri.host == 'unix/' then
+        -- Linux uses max 108 bytes for Unix domain socket paths, which means a
+        -- 107 characters string ended by a null terminator. Other systems use
+        -- 104 bytes and 103 characters strings.
+        local max_unix_socket_path = {linux = 107, other = 103}
+        local system = os.execute('[ $(uname) = Linux ]') == 0 and 'linux' or 'other'
```

Comment:
While we're here: isn't just `jit.os == 'Linux'` works?

Reply:
I guess it works, but it is not related to my changes :)

Reply:
And it is faster :)

OK, let's postpone.